mrtt-ui/src/components/ProjectDetailsForm/ProjectDetailsForm.js

```diff
@@ -38,68 +37,54 @@ function ProjectDetailsForm() {
         })
       )
       .min(1)
-      .typeError('Select at least one item')
+      .typeError('Select at least one country')
   })
   const formOptions = { resolver: yupResolver(validationSchema) }
 
   // get functions to build form with useForm() hook
-  const { register, control, handleSubmit, formState, watch } = useForm(formOptions)
+  const { control, handleSubmit, formState, watch } = useForm(formOptions)
   const { errors } = formState
   const watchHasProjectEndDate = watch('hasProjectEndDate', 'false')
+  const [isSubmitting, setisSubmitting] = useState(false)
+  const [isError, setIsError] = useState(false)
+
+  const onSubmit = async (data) => {
+    setisSubmitting(true)
+    setIsError(false)
+    let preppedData = []
+    const url = 'https://mrtt-api-test.herokuapp.com/api/v2/sites/1/registration_answers'
+
+    if (!data) return
 
-  const onSubmit = (data) => {
     // set up data structure for api
-    // POST to api route
-    console.log('data: ', data)
-  }
+    for (const [key, value] of Object.entries(data)) {
```

Comment:
As the app progresses, consider if this loop and its contents can be abstracted into a helper function that can be used in all forms

Reply:
good idea! will add this folder and think about how to abstract this